ccd-definition/FixedLists/CareSupervision/Supplements.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "Supplements",
```

Comment:
can it be in sync with java class, here it is Supplements, in java it is SupplementType. I would name it SupplementType

Reply:
renamed it as SupplementType